package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-promise": "^6.1.1",
     "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
+    "msw": "^2.2.1",
```

Comment:
I see you added msw but I don't see it being imported anywhere, was this accidental?

Reply:
I removed it. That was an accident.